ci/docker/python-wheel-manylinux.dockerfile

```diff
@@ -103,7 +103,11 @@ RUN vcpkg install \
 # Configure Python for applications running in the bash shell of this Dockerfile
 ARG python=3.8
 ENV PYTHON_VERSION=${python}
-RUN PYTHON_ROOT=$(find /opt/python -name cp${PYTHON_VERSION/./}-*) && \
+RUN if [[ ${PYTHON_VERSION} == "3.6" || ${PYTHON_VERSION} == "3.7" ]]; then \
```

Comment:
We're not compatible with 3.6 or 3.7 anymore, so this can probably be simplified?